moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1728,6 +1799,17 @@ bool PlanningScene::processCollisionObjectMsg(const moveit_msgs::CollisionObject
     }
     if (!object.type.key.empty() || !object.type.db.empty())
       setObjectType(object.id, object.type);
+    // Add named frames
+    std::map<std::string, Eigen::Affine3d> named_frames;
+    Eigen::Affine3d p;
+    for (std::size_t i = 0; i < object.named_frames.size(); ++i)
```

Comment:
Unfortunately, we can't update the messages in kinetic-devel without breaking the release. Can we target melodic with these changes.

Reply:
I am not sure anymore what the conclusion of our discussion regarding this was @davetcoleman. I don't have an issue with targeting only melodic.